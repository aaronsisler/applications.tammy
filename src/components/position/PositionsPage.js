import React from 'react';
import PositionContainer from '../position/PositionContainer';

export default class PositionsPage extends React.Component {
    constructor(props) {
        super(props);
    }

    componentDidMount() {
    }

    render() {
        return (
            <div id="positions_page">
                <PositionContainer />
            </div>
        );
    }
}
