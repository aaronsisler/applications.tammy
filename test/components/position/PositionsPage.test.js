import React from 'react';
import { shallow } from 'enzyme';
import PositionsPage from 'Position/PositionsPage';

describe('PositionsPage', () => {
    it('should render PositionsPage correctly', () => {
        const wrapper = shallow(<PositionsPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
