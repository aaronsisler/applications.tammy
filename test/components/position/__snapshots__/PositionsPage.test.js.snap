// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PositionsPage should render PositionsPage correctly 1`] = `
<div
  className="positions_page"
>
  <Connect(PositionsContainer) />
</div>
`;
